src/helpers/utils.ts

```diff
@@ -175,9 +175,10 @@ export function filterProposals(space, proposal, tab) {
 
 export function infiniteScroll() {
   window.onscroll = () => {
+    console.log(window.screen.height);
     const bottomOfWindow =
-      document.documentElement.scrollTop + window.innerHeight ===
-      document.documentElement.offsetHeight;
+      document.documentElement.scrollTop + window.innerHeight >=
+      document.documentElement.offsetHeight - 100;
```

Comment:
This could be a parameter in the infiniteScroll function we can call it `distance` like on the infinity scroll module we were using with Vue2.

Reply:
When would we want to change this value?

Reply:
Probably never, but ideally we want to create UI components and functions generic and flexible enough that we can create a lib and reuse them in others codebase.

Reply:
Ok